sound/soc/codecs/max98373.h

```diff
@@ -212,6 +223,9 @@ struct max98373_priv {
 	bool interleave_mode;
 	unsigned int ch_size;
 	bool tdm_mode;
+	/* cache for reading a vaild fake feedback value */
```

Comment:
'vaild' may be misspelled - perhaps 'valid'?

Also there are warnings from checkpatch - "WARNING: line over 80 characters"

Reply:
> 'vaild' may be misspelled - perhaps 'valid'?

Fixed, thanks.

> 
> Also there are warnings from checkpatch - "WARNING: line over 80 characters"

The limitation is 100 characters now. So, it should be fine.